### PR TITLE
AC Automated Test - LRAC-10264 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10264

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACEventAnalysis.path [hunk 1]
```diff
@@ -55,6 +55,11 @@
 	<td>//span[text()='${key_informationValue}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>VIEW_DASHBOARD_INFORMATION</td>
+	<td>//div[@class='attribute-value'and text()='${key_attributeName}']</td>
+	<td></td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACEventAnalysis.path:61]: hi @ViniLopes20 , could you please add between the class name and the `and`?

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACEventAnalysis.path [hunk 2]
```diff
@@ -60,6 +60,11 @@
 	<td>//div[@class='attribute-value'and text()='${key_attributeName}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHOOSE_FILTER</td>
+	<td>//div[contains(@class,'show')]//button[contains(@class,'button-root')and text()='${key_filterName}']</td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACEventAnalysis.path:65]: hi @ViniLopes20 , could you please add a space after the class and the `and` ?

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**